guarddog/analyzer/sourcecode/dll-hijacking.yml

```diff
@@ -0,0 +1,69 @@
+rules:
+  - id: dll-hijacking
+    languages:
+      - python
+    message: This package manipulates a trusted application into loading a malicious dll
+    metadata:
+      description: Identifies when a trusted application is manipulated into loading a malicious dll
```

Comment:
Would suggest updated wording to avoid using passive voice:

> Identifies when a malicious package manipulates a trusted application into loading a malicious dll